backend/README.md

```diff
@@ -0,0 +1 @@
+to generate python classes for the messages and the service in the proto file run the build.sh script
```

Comment:
I assume we are going to submit generated files (at least for now)?

Reply:
yes they are located in the grpc/messages folder